### PR TITLE
ci pipeline

### PR DESCRIPTION
**Pull request by cristianciutea**
This PR will:
- revert to maven build, because jlink can cause dependencies issues on connectors
- fix failing tests
- fix existing linux build pipeline
- create Windows build pipeline
- create GH workflows
- remove jmxterm

### CODE REVIEW

File: .github/workflows/prerelease.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Go
```
Comment (rubenruizdegauna) [inline .github/workflows/prerelease.yml:34]: go is not necessary, isn't it?

File: pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>nrjmx</groupId>
     <artifactId>nrjmx</artifactId>
-    <version>1.4.2</version>
+    <version>0.0.0</version>
```
Comment (rubenruizdegauna) [inline pom.xml:7]: is it ok the version ☝️

Comment (cristianciutea) [reply pom.xml:7]: we inject the version on build, so no need to update this line everytime

**Review by rubenruizdegauna — APPROVED** [review]
LGTM

**Pull request merged**